frontend/javascript/components/campaign-recommend.vue

```diff
@@ -55,12 +55,21 @@
 
     <div class="text-center">
       <a
-        class="btn btn-info sharing__link -twitter"
+        class="btn btn-info sharing__link -bluesky"
```

Comment:
That class looks kinda broken. Better to replace any custom CSS with froide's `btn-share-bluesky`.